bundles/org.openhab.core.config.discovery.sddp/src/test/java/org/openhab/core/config/discovery/sddp/test/SddpDiscoveryTests.java

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.core.config.discovery.sddp.test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Optional;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.junit.jupiter.api.Test;
+import org.openhab.core.config.discovery.sddp.SddpDevice;
+import org.openhab.core.config.discovery.sddp.internal.SddpDiscoveryService;
+
+/**
+ * JUnit tests for parsing SDDP discovery results.
+ *
+ * @author Andrew Fiddian-Green - Initial contribution
+ */
+@NonNullByDefault
+public class SddpDiscoveryTests {
+
+    private static final String ALIVE_NOTIFICATION = "NOTIFY ALIVE SDDP/1.0\r\n" +
+    // @formatter:off
+            "From: \"192.168.4.237:1902\"\r\n" +
+            "Host: \"JVC_PROJECTOR-E0DADC152802\"\r\n" +
+            "Max-Age: 1800\r\n" +
+            "Type: \"JVCKENWOOD:Projector\"\r\n" +
+            "Primary-Proxy: \"projector\"\r\n" +
+            "Proxies: \"projector\"\r\n" +
+            "Manufacturer: \"JVCKENWOOD\"\r\n" +
+            "Model: \"DLA-RS3100_NZ8\"\r\n" +
+            "Driver: \"projector_JVCKENWOOD_DLA-RS3100_NZ8.c4i\"\r\n";
+    // @formatter:on
```

Comment:
Maybe you can use Java text blocks in this file? 

https://www.baeldung.com/java-text-blocks

Reply:
@wborn that's cool!

Reply:
Done

Reply:
Nice :slightly_smiling_face: